3-appfactory/modules/app-group-baseline/main.tf

```diff
@@ -55,20 +56,25 @@ module "tf_cloudbuild_workspace" {
   project_id               = module.app_admin_project.project_id
   tf_repo_uri              = google_sourcerepo_repository.app_infra_repo.url
   tf_repo_type             = "CLOUD_SOURCE_REPOSITORIES"
-  artifacts_bucket_name    = "${var.application_name}-build-${module.app_admin_project.project_id}"
-  create_state_bucket_name = "${var.application_name}-state-${module.app_admin_project.project_id}"
-  log_bucket_name          = "${var.application_name}-logs-${module.app_admin_project.project_id}"
+  location                 = var.location
+  trigger_location         = var.trigger_location
+  artifacts_bucket_name    = "${var.bucket_prefix}-${module.app_admin_project.project_id}-${var.application_name}-build"
+  create_state_bucket_name = "${var.bucket_prefix}-${module.app_admin_project.project_id}-${var.application_name}-state"
```

Comment:
Esse **state bucket** pode usar o do bootstrap depois que o PR for mergeado?

Reply:
if it makes sense yes, we could replace with the value from bootstrap